packages/utilities/psammead-storybook-helpers/README.md

```diff
@@ -4,9 +4,18 @@ This package provides a collection of common values that are used in storybook b
 
 ## Exports
 
-`LANGUAGE_VARIANTS` - A list of text samples in different languages.
+`LANGUAGE_VARIANTS` - A list of text samples in different languages, with the script and direction that should be used for that language.
 
-`inputProvider` - A function that provides support for selecting between the `text` and `select` storybook knobs. When using the `select` knob, users will be prompted to select a language, which will insert text from `LANGUAGE_VARIANTS` into the story.
+`inputProvider` - A function that provides support for previewing components in storybook in different languages. Takes two arguments, `slots` and `renderFn`. Sets the `dir` attribute on the `<html>` element in the story iframe using [Helmet](https://www.npmjs.com/package/react-helmet).
+
+- `slots`: `[{ name, defaultText }]` Required (but can be empty or null).
+  - `name`: String uniquely identifying this slot in the story. Required.
```

Comment:
`name` isn't strictly "required" if `slots` can equal `null`

Reply:
it's required in the sense that `[{defaultText: 'something'}]` will break while `[{name: 'a name'}]` won't. It's probably badly worded and I'll try and come up with something better